### ISSUE TITLE
Unable to launch EA Origin games when STEAM_LINUX_RUNTIME_LOG=1

### ISSUE THREAD

**Issue by Cat-Lady** [ValveSoftware/steam-runtime#402]
* Steam Runtime Version: 
```
#Name	Version		Runtime	Runtime_Version	Comment
SteamLinuxRuntime	v0.20210317.0-0-ge09b2e6			# Entry point scripts, etc.
pressure-vessel	0.20210415.0+srt1	scout	0.20210415.0	# pressure-vessel-bin.tar.gz
soldier	0.20210415.0	soldier	0.20210415.0	# com.valvesoftware.SteamRuntime.Platform-amd64,i386-soldier-runtime.tar.gz
```

* Distribution:
Debian 10+
 
* Link to your full system information: 
[systeminfo.log](https://github.com/ValveSoftware/steam-runtime/files/6362441/systeminfo.log)

* Are you using the Steam Linux Runtime compatibility tool?:
Soldier - Yes, Scout - No.

#### Please describe your issue in as much detail as possible:

Hello Steam developers,

1. When trying to gather logs from Steam Runtime (soldier) for debugging unrelated issue, it turned out that having ``STEAM_LINUX_RUNTIME_LOG=1`` set before launching Steam maxes it unable to (successfully) launch games that use EA Origin middleman. Specifically, subsequently launched Origin client returns nonsense about game being not installed (each and every time.)

2. Attempting to run the game again, without the env variable set, starts it successfully.

3. I've tested it with "EA Star Wars: Battlefront II" and "Star Wars: Fallen Order". I suppose it might be the case with each game run through Origin middleman.

Relevant log from the Steam Runtime, from the failed attempt:
[origin_fail_slr-app1237950-s1021efbfe445190c.log](https://github.com/ValveSoftware/steam-runtime/files/6362457/origin_fail_slr-app1237950-s1021efbfe445190c.log)

...and one from Proton:
[steam-1237950.log](https://github.com/ValveSoftware/steam-runtime/files/6362463/steam-1237950.log)

**Comment by RyuzakiKK** [comment]
I'm still not sure why this is happening, but I can confirm that I'm able to reproduce it with the free to play game "The Sims™ 4 Create A Sim Demo" (1335960).

It seems like the runtime's output log redirect is somehow interfering with Origin.

**Comment by Ryochan7** [comment]
![good_grumpy_cat](https://user-images.githubusercontent.com/581968/115900669-d6d59980-a425-11eb-8686-f7b9d4065e3c.jpeg)

**Comment by smcv** [comment]
This should be fixed in the next beta, versioned 0.20210428.0 or later. To resolve this sooner, you could try replacing `SteamLinuxRuntime_soldier/_v2-entry-point` with [the development version](https://gitlab.steamos.cloud/steamrt/steamlinuxruntime/-/blob/master/common/_v2-entry-point).

Or, as a workaround: avoid using `STEAM_LINUX_RUNTIME_LOG` with Origin games.

**Comment by smcv** [comment]
This is believed to be fixed in `SteamLinuxRuntime_soldier` version 0.20210430.0, currently the `client_beta` branch.

For this particular issue, the important line in `VERSIONS.txt` is `SteamLinuxRuntime v0.20210430.0-1-gf204d22` or later.

**Comment by kisak-valve** [comment]
Closing per the last comment.
